kuma/actioncounters/migrations/0003_update_unique_hashes.py

```diff
@@ -16,8 +16,8 @@ def forwards(self, orm):
         # but wasn't implemented before this migration. Doing it now, so the
         # data migration has a smaller set to work with.
         (orm.ActionCounterUnique.objects
-                .filter(user=None, modified__lt=datetime.datetime(2011,8,1))
-                .delete())
+            .filter(user=None, modified__lt=datetime.datetime(2011, 8, 1))
+            .delete())
 
         # Update all remaining counters with a unique hash.
         from actioncounters.utils import get_unique
```

Comment:
@jezdez I had to change this line to `from kuma.actioncounters.utils import get_unique` to make migrations run during vagrant provision. Is that a safe change to make permanent in `master` branch?